Documentation/building/cross-platform-testing.md

```diff
@@ -27,7 +27,7 @@ instructions assume you are building for Linux, but are easily modifiable for OS
    CoreFX on Linux via `build.sh`. 
    The other option is:
 
-   * Build the managed parts of CoreFX on Windows. To do so run `build-managed.cmd -os=Linux -target-os=Linux`. It is okay to build a Debug version of CoreFX and run it
+   * Build the managed parts of CoreFX on Windows. To do so run `build-managed.cmd -os=Linux`. It is okay to build a Debug version of CoreFX and run it
```

Comment:
FYI, running build-managed.cmd from a clean build, now fails because the native bits will be missing and packaging step will fail trying to find them.  I'll create an issue tracking this.